adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -41,6 +41,114 @@ import scala.collection.JavaConversions._
 private[adam] class FastqRecordConverter extends Serializable with Logging {
 
   /**
+   * Parse 4 lines at a time
```

Comment:
This doc comment doesn't match the method.

Perhaps something like `Return true if the read name suffix and flags match.`